packages/lesswrong/components/ea-forum/SmallpoxBanner.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { createStyles } from '@material-ui/core/styles'
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import { SECTION_WIDTH } from '../common/SingleColumnSection';
+import { cloudinaryCloudNameSetting } from '../../lib/publicSettings';
+import { Link } from '../../lib/reactRouterWrapper';
+
+const bannerHeight = 250
+const mobileImageId = 'Banner/Smallpox_Eradication_Day-17.png'
+const desktopImageId = 'Banner/Smallpox_Eradication_Day-16.png'
+const featuredPost = '/posts/jk7A3NMdbxp65kcJJ/500-million-but-not-a-single-one-more'
+
+const mobileImage = `https://res.cloudinary.com/${cloudinaryCloudNameSetting.get()}/image/upload/w_${SECTION_WIDTH},h_${bannerHeight}/${mobileImageId}`
+const desktopImage = `https://res.cloudinary.com/${cloudinaryCloudNameSetting.get()}/image/upload/w_${SECTION_WIDTH},h_${bannerHeight}/${desktopImageId}`
+
+const styles = createStyles((theme: ThemeType): JssStyles => ({
+  link: {
+    '&:hover': {
+      opacity: 'unset'
+    }
+  },
+  image: {
+    objectFit: 'cover'
```

Comment:
I'm slightly suspicious of this (I'm not sure why it doesn't break layout on mobile adding a horizontal scrollbar, but I guess there's some ancestor component that has `overflow: hidden` set). Seems OK for now...

Reply:
You think I should add `overflow: hidden` as well to be safe?

Reply:
Yeah, I think it's probably fine, although I have some taste sense that it's not great to have elements falling off the screen in a somewhat uncontrolled way. Not sure what this'll do e.g. in Edge